js/my.js

```diff
@@ -232,3 +232,26 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  cmd = cmd.replace(/^\//, '');
+
+  var args = cmd.match(/(\S+) ?(\S+)? ?(.*)/);
```

Comment:
@nickdesaulniers I only want to split the first two off, and keep the rest in a single variable. I can't think of any cleaner way - what do you suggest?